config/database.php

```diff
@@ -65,7 +65,7 @@
         'mariadb' => [
             'driver' => 'mariadb',
             'url' => env('DB_URL'),
-            'host' => env('DB_HOST', '127.0.0.1'),
+            'host' => env('DB_HOST', 'db'),
```

Comment:
```suggestion
            'host' => env('DB_HOST', '127.0.0.1')
```
این تغییر تاثیری نداره چون مقدار از env خونده میشه. اگر در env نبود، مقدار پیشفرض میشه db یا 127.0.0.1